### ISSUE TITLE
VirusTotal says it's insecure

### ISSUE THREAD

**Issue by ChaoticUltimateJ** [eldewrito2/ElDewritoLauncher#7]
[Here's the link to the results](https://www.virustotal.com/gui/file/7ce2de9987684a505de05a4ea5b21b80b0d1514ff1b66af72353a1de773eb61a)
Screenshot ~~for the lazy.~~
![image](https://github.com/eldewrito2/ElDewritoLauncher/assets/76010200/b0ee09a4-eb10-4e03-88b1-c7fa103beedd)

**Comment by inter0hm** [comment]
Virustotal is not meant to be used for virus analysis people uses our tool ( malware researcher) without understanding them. Those vendor are some that people rarely trust detection.  It's mostly used for crowdsourcing, matching exe with their traffic, domain and intelligence research.  The only reasons most of game you play do not get detected is because they benefit from whitelisting. Alot of game will use packer in order to prevent client decompilation of their game so hacker have harder time getting valuable data ( im over simplifying it ) .
Two often launcher are wrongly detected  as file dropper, nothing is wrong about the launcher. You can look at the code, compile it yourself. 
You're simply scaring yourself with data you do not understand, leave such to people that actually understand it and work in this field.

**Comment by ChaoticUltimateJ** [comment]
> Virustotal is not meant to be used for virus analysis people uses our tool (malware researcher) without understanding them. Those vendor are some that people rarely trust detection. It's mostly used for crowdsourcing, matching exe with their traffic, domain and intelligence research. The only reasons most games you play do not get detected is because they get whitelisted. A lot of games will use a packer in order to prevent client decompilation of their game so hackers have a harder time getting valuable data (I'm over simplifying it).
> Very often launchers are wrongly detected as file droppers, nothing is wrong about the launcher. You can look at the code and compile it yourself. 
> You're simply scaring yourself with data you do not understand, leave such to people that actually understand it and work in this field.

It's been about a year since I opened this issue lmao. I've learned a lot more about cyber security since posting this. Sure, I'm not an expert and most certainly never will be, but I'm not scared to run something that has its source code publicly available.

As for "leaving this to people who actually understand", I do understand VirusTotal is not the best place to test for a virus, that's what I have a second computer and common sense for of course.

Also, I fixed your grammar because I felt like it, though some of your points are actually wrong; such as the preventing "client decompilation" as most people after game data will use the game installed, not data from an installer; and hackers will usually do the same thing, modifying an existing install rather than editing the installer; and, again, the source code is publicly available here, so it wouldn't be difficult for a skilled hacker to compile their own version with cheats or something.

Also-also, VirusTotal works by comparing file hashes to see if two files are the exact same, if there's a match then it gives the same result every time, meaning two files that just so happen to have the same file hash (often referred to as hash collision, though this is an event that is highly unlikely) will give bad results.

I don't trust VirusTotal as some all-in-one, foolproof, perfect antivirus, common sense and a second computer are all anyone needs to prevent something bad from happening.

TL;DR, To heavily over-simplify, I opened this issue when I was an idiot, and am now replying to your comment on it as an informed fellow intellectual who won't make this same mistake again.
